FamilySearchIosSampleApp/TreeTVC.swift

```diff
@@ -0,0 +1,249 @@
+//
+//  TreeVC.swift
+//  FamilySearchIosSampleApp
+//
+//  Created by Eduardo Flores on 6/6/16.
+//  Copyright © 2016 FamilySearch. All rights reserved.
+//
+
+import UIKit
+
+class TreeTVC: UITableViewController {
+    
+    @IBOutlet weak var activityIndicator: UIActivityIndicatorView!
+    
+    var user : User!
+    
+    var personArray = NSArray()
+    
+    var accessToken : String!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        // get the access token from NSUserDefaults
+        let preferences = NSUserDefaults.standardUserDefaults()
+        accessToken = preferences.stringForKey(Utilities.KEY_ACCESS_TOKEN)
+        
+        // get url for family tree from Collections
+        Utilities.getUrlsFromCollections({ (collectionsResponse, error) -> Void in
```

Comment:
Hi Stephen,

I agree with you that this would make no sense on a release/production app. However, when we started planning it for Android we (Matt and I) decided that since this app is for demo purposes for developers, it might make more sense for developers to see all of the elements of each "section" (Login section, ancestor details section, etc...) all together within the same view controller.
I agree, it is a slow process and it will be horrible for end users to have something like this workflow, but it is much easier for developers to just gather the specific data, for the specific section they want.

What are your thoughts on this?